robotbase/src/main/java/frc/robot/commands/drive/DriveToGamepeice.java

```diff
@@ -14,12 +14,12 @@ public class DriveToGamepeice extends Command {
   private Pose2d m_initialPose;
 
   public DriveToGamepeice() {
-    addRequirements(Robot.swerve, Robot.shooterCam);
+    addRequirements(Robot.swerve);
```

Comment:
Should require the intake cam here to prevent other commands from doing things like switching pipelines while this command is running.